README.md

```diff
@@ -88,6 +89,22 @@ To enable this functionality, a user should be set up on GitLab, with GitLab 'De
         * Set *Name of the repository" to ``origin`` 
         * Set *Branch to merge* as ``${gitlabTargetBranch}``
 
+#### GitLab >= 8.1
+1. In the *Source Code Management* section:
+    1. Click *Git*
+    2. Enter your *Repository URL* (e.g.: ``git@your.gitlab.server:group/repo_name.git``)
+      * In the Advanced settings, set its *Name* to ``origin`` and its *refspec* to ``+refs/heads/*:refs/remotes/origin/* +refs/merge-requests/*/head:refs/remotes/origin/merge-requests/*``
+    3. To be able to merge from forked repositories:  <br/>**Note:** this requires [configuring communication to the GitLab server](#configuring-access-to-gitlab)
+      * Add a second repository with:
+        * *URL*: Your *Repository URL* as in the previous step (e.g.: ``git@your.gitlab.server:group/repo_name.git``)
+    4. In *Branch Specifier* enter:
+      * ``merge-requests/${gitlabMergeRequestIid}``
```

Comment:
Not a typo. GitLab uses both `id` and `iid`. `id` refers to the internal ID in their DB, and `iid` to the user facing number (in tickets or Merge Requests). In this case, we need the user facing one.